solutions/bronze/usaco-1276.mdx

```diff
@@ -0,0 +1,76 @@
+---
+id: usaco-1276
+source: USACO Bronze 2023 January
+title: Air Cownditioning II
+author: Jesse Choe
+---
+
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+vector<pair<int, int>> cowlocs;
+vector<int> comfort;
+vector<pair<int, int>> conditioners;
+vector<int> cost;
+vector<int> power;
+int ans = 1e9, n, m;
+
+int check(int subset) {
```

Comment:
what this do

Reply:
it's from the official analysis... i just changed the way subsets are generated